packages/devtools_shared/lib/src/server/server_api.dart

```diff
@@ -32,7 +33,8 @@ class ServerApi {
   /// To override an API call, pass in a subclass of [ServerApi].
   static FutureOr<shelf.Response> handle(
     shelf.Request request,
-    ExtensionsManager extensionsManager, [
+    ExtensionsManager extensionsManager,
+    DeeplinkManager deeplinkManager, [
     ServerApi? api,
   ]) {
```

Comment:
This will be a breaking change: https://github.com/dart-lang/sdk/blob/main/pkg/dds/lib/src/devtools/handler.dart#L134.

If we need to break this anyway, perhaps we should consider switching this signature to use named parameters so that we don't have to deal with more breaking changes in the future.

```
  static FutureOr<shelf.Response> handle(
    shelf.Request request, {
    required ExtensionsManager extensionsManager,
    required DeeplinkManager deeplinkManager,
    ServerApi? api,
  }) {
```

And then we'll need to do a major version bump for this package to ^4.0.0